modules/ROOT/pages/import-asset-exchange-task.adoc

```diff
@@ -13,6 +13,32 @@ Additionally you can click Dependency Snippets to see the artifact's information
 
 Anypoint Studio downloads and imports the asset automatically.
 
+[[import-asset-outside-studio]]
+== Manually Import an Asset from Exchange
+
+If you are having trouble accessing Exchange from Studio 6, follow the following steps to import an asset from Exchange from outside Studio:
```

Comment:
```suggestion
If you are having trouble accessing Exchange from Studio 6, use the following steps to import an asset from Exchange from outside Studio:
```

Reply:
" . . . follow the following . .." seems a bit repetitive - I suggested "use" but "perform" or "complete" would probably also work